### ISSUE TITLE
[FEATURE]  use previous computation

### ISSUE THREAD

**Issue by arita37** [darioizzo/dcgp#96]
Just wondering, if there is a way to use previous formulaes
as starting points for evolution ?

Mostly, load as initial population (vs starting from scratch every time...),
Thanks

**Comment by darioizzo** [comment]
You can feed them into the pagmo population as chromosomes?

**Comment by arita37** [comment]
Thx.

Goal is to find a way to inject 
either previous computation as initial seed
or some human formulae as initial seed

to prevent from starting from scratch again
every time.

Thz

**Comment by darioizzo** [comment]
Yes, understood. At the end of an evolution you can save the best chromosome (i.e. a formula) of the population.

Then you can create at any point an empty population and push-back that chromosome in togheter with any other random or evolved formula.

At that point you can continue evolving.

**Comment by arita37** [comment]
Sure, thanks

Is there any simple example available ?
(where seed is custom injected).

thanks

**Comment by darioizzo** [comment]
Unfortunately not. Its going to be something like this ... 
```
x = previosly computed seed
pop = pg.population(sr_prob,3) # creates a population with 3 random individuals
pop.push_back(x) # now the population has 4 individuals, one is the seed
pop = algo,evolve(pop)
```
 but `sr_prob` must have the same dimensions and kernels as the one that created `x`.

As per your question on the possibility to use a human formula as seed ... there is no utility atm that creates a chromosome from a formula. This is not difficult per se, but there are infinite ways of doing so (i.e. infinite chromosomes that correspond to the same formula) so one would have to take some decisions there, which will also affect the evolution afterwards.
